### PR TITLE
issue169 Include release notes for RC1

### PR DESCRIPTION
**Pull request by mmusgrov**
#169  Release Candidate
We are missing some release notes.
I also updated the arquillian dependency so that we can depend on the pre-existing [CQ 13755](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=13755)

### CODE REVIEW

**Comment by mmusgrov** [comment]
@xstefank @ochaloup @rdebusscher @tomjenkinson I think anyone can review these release notes (ie I don't think we need to wait for Rudy)

File: spec/src/main/asciidoc/release_notes.asciidoc [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    <groupId>org.eclipse.microprofile.lra</groupId>
+    <artifactId>microprofile-lra-api</artifactId>
+    <version>${version.microprofile.lra}</version>
+    <version>1.0</version>
+    <scope>provided</scope>
+</dependency>
+----
+
+Create a JAX-RS business resource and annotate the methods that you would like to be included in a long running action
+using the @LRA annotation. Mimally you should define which business method that should be run if the LRA is cancelled
```
Comment (xstefank) [inline spec/src/main/asciidoc/release_notes.asciidoc]: typo Minimally, I would also suggest dropping "that"

File: spec/src/main/asciidoc/release_notes.asciidoc [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+using the @Compensate annotation.
+
+[source,java]
+----
+public class SimpleParticipant {
```
Comment (xstefank) [inline spec/src/main/asciidoc/release_notes.asciidoc]: Not necessary, but we can add Path annotation also on class, just to make it clear this is a JAX-RS resource. Up to you if you want to update it.

File: spec/src/main/asciidoc/release_notes.asciidoc [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    public Response businessOp(@HeaderParam(LRA_HTTP_RECOVERY_HEADER) URI recoveryId,
+                                    @HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId) {
+        // perform some business action
+        return Response.ok().build();
+    }
+
+    @PUT
+    @Path("/compensate")
+    @Compensate
+    public Response compensateWork(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId, String userData) {
```
Comment (xstefank) [inline spec/src/main/asciidoc/release_notes.asciidoc]: userData was postponed to 1.x - https://github.com/eclipse/microprofile-lra/issues/15

File: spec/src/main/asciidoc/release_notes.asciidoc [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        // compensate for any actions perform
```
Comment (xstefank) [inline spec/src/main/asciidoc/release_notes.asciidoc]: typo "performed actions"

**Comment by mmusgrov** [comment]
The instructions for the release notes are https://wiki.eclipse.org/MicroProfile/ReleaseNotes.
 
@xstefank @ochaloup @rdebusscher @tomjenkinson I think anyone can review these release notes (ie I don't think we need to wait for Rudy)

**Pull request merged**